Part2_Mapplots/ex2_2_psyplot.ipynb

```diff
@@ -193,9 +193,10 @@
     "ax = plt.axes(projection=ccrs.Robinson())\n",
```

Comment:
<p>How big is the dataset? Can we not add that netCDF file to the repo? With ftp there is always a danger that the file might not exist at a later point and one cannot use the material offline.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/C2SM/pyvis/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
- my_exp1_atm_3d_ml_20180921T000000Z_t2m.nc is **588KB**
- rmp_my_exp1_atm_3d_ml_20180921T000000Z.nc is **184KB**
- cosmo_pollen.nc is **214MB** (only needed for the user examples)

Should I add the first two as they are part of an exercise?

Reply:
<p>Yes, please add the first two to the repo</p>